src/platform/user/profile/actions/index.js

```diff
@@ -30,6 +31,62 @@ export function profileLoadingFinished() {
   };
 }
 
+export function getUserData(dispatch) {
+  fetch(`${environment.API_URL}/v0/user`, {
+    method: 'GET',
+    headers: new Headers({
+      Authorization: `Token token=${sessionStorage.userToken}`
+    })
+  }).then(response => {
+    if (response.ok) return response.json();
+    const error = new Error(response.statusText);
+    error.status = response.status;
+    throw error;
+  }).then(json => {
+    const userData = json.data.attributes.profile;
+    // sessionStorage coerces everything into String. this if-statement
+    // is to prevent the firstname being set to the string 'Null'
+    if (userData.first_name) {
+      sessionStorage.setItem('userFirstName', userData.first_name);
+    }
+    // Report out the current level of assurance for the user
+    recordEvent({ event: `login-loa-current-${userData.loa.current}` });
+    dispatch(updateProfileFields({
```

Comment:
I think this action creator only uses one argument.

Reply:
Not sure what you mean. I think we're only passing one argument to `updateProfileFields`